model/db/mongo/mongo.php

```diff
@@ -701,6 +701,8 @@ public function getRef()
 	 */
 	public function getRow()
 	{
+		debug_assert( false, 'Function getRow is scheduled for deletion, replace with Model::getById( $id )' );
```

Comment:
That this function were scheduled for deprecation ( as i already informed you before ), but instead of contacting about deprecation reasons to possibly cancel them, it was just silently removed. This is not a correct way to do stuff.

I've spoken with @tapiau after my comment though, and it turned out that getRow is used in model constructor which is used everywhere in atrium, so i cancel this deprecation initiative, as i cannot judge that as safe operation anymore.

Long things short, getRow() is not deprecated anymore, so assert should be removed.

Ps. It would be really useful if @tapiau were not the only one in your project who has assertions enabled. It's very unsafe to have them disabled on dev, but it's up to you.